Utilities/File.cpp

```diff
@@ -370,23 +376,400 @@ namespace fs
 		return false;
 	}
 
-	bool device_base::remove(const std::string&)
-	{
-		g_tls_error = error::readonly;
-		return false;
-	}
+	bool device_base::remove(const std::string&)
```

Comment:
why are there a billion seemingly unnecessary changes in this file?

Reply:
Because you guys do not use clang-format, but it is configured for project. Looks like you @Megamouse do job manually instead of code format tool :D

Reply:
we do. but it's a bug in clang-format.
we don't want the extra indent in these cases

Reply:
OK, for this file issue not related with clang-format. 2 classes was moved out from constructor, github diff fails to understand it

Reply:
I suggest putting unix_file and windows_file in a new file (maybe native_file.h)

Reply:
> we don't want the extra indent in these cases

Well,
```c
// clang-format off
                          void       MyCode_Means__My_____RULES( void * *
                          **MY_pointer
){    }
// clang-format on
```
should do a thing :smile: Btw, bug with indents in lambdas was fixed long time ago

     
> I suggest putting unix_file and windows_file in a new file (maybe native_file.h)

OK, I hope diff tools are smart enough in this case to understand that code was moved, not written by me

Reply:
The bug exists in combination with another setting we use if I remember correctly.
Doesn't matter though